### DISCUSSION TITLE
Larger pads for certain components

### DISCUSSION THREAD

**Discussion by mDubovsky** [pilotak/eth-io#11] [Ideas]
The TLP290-4 was a bit hard to solder with a soldering iron due to the leads obscuring much of the soldering pads.
It was almost impossible to solder the inductor in the 5V supply as the inductor covers the SMD pads completely.
It would be beneficial to edit footprint of these components and enlarge the pads for easier assembly.

**Comment by pilotak** [comment]
I have to agree that the inductor is a bit harder to hand solder *(i just got used to it)*, the footprint is for reflow soldering. I will adjust it

**Comment by pilotak** [comment]
done in 28b9ccb64864791d090ab6205445d9660984af67
